examples/forms.amp.html

```diff
@@ -238,15 +238,12 @@
     </fieldset>
 
     <div submit-success>
-        <template type="amp-mustache">
-            Success! Thanks {{name}} for subscribing! Please make sure to check your email {{email}}
-        to confirm!
-        </template>
+      Success! Thanks for subscribing! Please make sure to check your email
```

Comment:
Hmm it might be clearer to have this example without templates on both, so a reader would see that a form doesn't need any template tag to work correctly, as the user in #5619 expected. I'm fine adding a more appropriate `amp-img` for the use-case though.

If you definitely want an example with `submit-success` having a template and `submit-error` not having a template, I can change another one of the forms below this one.